server/sonar-server/src/main/java/org/sonar/server/activity/ActivityService.java

```diff
@@ -45,8 +46,15 @@ public void save(Activity activity) {
       .setAction(activity.getAction())
       .setMessage(activity.getMessage())
       .setData(KeyValueFormat.format(activity.getData()))
+      .setProfileKey((String) activity.getData().get("profileKey"))
```

Comment:
I will keep the information in the data field for the time being (because of impacts), but I add a new property profileKey in Activity